### DISCUSSION TITLE
"No method for:" in command line shell

### DISCUSSION THREAD

**Discussion by mmpadilla15** [thegetty/quire#474] [Get Help] [answered]
I couldn't find this issue in the discussion ... I'm trying to output my project so I can review the downloadables and make any edits. I've typed in `quire site` and `quire preview` but for both commands, I get a "No method for:" response.

![Screen Shot 2022-09-02 at 12 32 43 PM](https://user-images.githubusercontent.com/74433007/188207788-918f8931-790a-4725-a023-8b27db827294.png)

I'm not sure what I'm doing wrong. Can someone advise?

**Comment by geealbers** [comment, marked as answer]
Hi @mmpadilla15 it looks like you also have `quire preview` running while trying those other commands. They should be run independently.  Stop the preview first, and then try `quire site` or `quire epub`, and let us know if that does the trick. 🤞

**Reply by mmpadilla15** [reply]
Hi @geealbers! I was able to get the `quire site` to work, but not `quire epub` or `quire mobi`. I get an error about installing Pandoc:

![Screen Shot 2022-09-06 at 12 52 32 PM](https://user-images.githubusercontent.com/74433007/188705413-735aa8a0-9bd2-40ff-a034-75575eed7dae.png)

I feel like I've read about this somewhere but I can't recall where. Can you advise on that?

**Reply by geealbers** [reply]
Yes, Pandoc mentioned in our "Full Install" instructions in the documentation. You can get the Mac installer here: https://pandoc.org/installing.html

**Reply by mmpadilla15** [reply]
@geealbers Found it - thank you! And I've installed it. But now I'm getting this error when I run `quire epub` and `quire mobi`:

<img width="660" alt="Screen Shot 2022-09-07 at 3 25 38 PM" src="https://user-images.githubusercontent.com/74433007/188971053-1a364637-3eaf-4c7d-a496-4ac5637a1dff.png">

If I understand it correctly, there's an error in one of my md files at line 15. But I'm not seeing the error:

<img width="915" alt="Screen Shot 2022-09-07 at 3 26 35 PM" src="https://user-images.githubusercontent.com/74433007/188971206-a2aac385-9919-4d33-92ab-e606c7fa16fe.png">

I was thinking it might be the colon in the exhibition title, but I'm using that exact text on other plates as well.

**Reply by Erin-Cecele** [reply]
Hi @mmpadilla15 I believe the error is related to the fact that you have a closing q-class shortcode but not an open one. That's why it says "got closing shortcode, but none is open."  You either need to remove {{< /q-class >}} or add an opening shortcode above the exhibition information.

**Reply by mmpadilla15** [reply]
@Erin-Cecele Got it! And they are all working now.
